protocol/app/stoppable/stoppable_test.go

```diff
@@ -0,0 +1,41 @@
+package stoppable
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+type MockStoppable struct {
```

Comment:
nit: `MockStoppable` -> `FakeStoppable` or `TestStoppable`. Lets reserve the word mock for things that are actually generated code that have the typical mock methods.